share/dictionary/der/dictionary.common

```diff
@@ -0,0 +1,55 @@
+DEFINE GeneralName      choice
+BEGIN GeneralName
+
+ATTRIBUTE otherName     0       sequence option=0
```

Comment:
run `./scripts/dict/format.dl share/dictionary/der/dictionary*`

for consistent formatting